sprint-planning/readme.md

```diff
@@ -1,29 +1,72 @@
 # Planning
 
-## Content TODO's
+## Goals
 
-- Complete outline below
-- Determining team capacity
-- Tasking sprint stories
-  - size of tasks
-- WIP limits / dividing up into workstreams
+During the [sprint planning](https://www.agilealliance.org/glossary/sprint-planning), the team discusses and agrees on the scope for the upcoming sprint. 
 
-## Goals
+**Goals are:**
+- Selecting the **stories** that will be implemented in the sprint.
+- Estimating the **effort** required for the stories in the sprint.
+- Splitting the stories into **tasks**.
 
-What are the intended outcomes of the ceremonies?
+General guidance:
+- The sprint planning should happen at the begining of the new sprint (or at the end of the previous one). It usually last bewteen 1 and 4 hours depending on the size of the team and duration of the sprints.
+- Each story should be achievable in the sprint. Otherwise, you should split the story.
+- Each task duration should be somewhere between 2 to 8 hours.
 
 ## Participation
 
-What members of the team should participate?
+**Everyone** in the team should participate in the sprint planning.
+
+**Specific roles:**
+- **[Scrum Master](https://www.agilealliance.org/glossary/scrum-master)**: 
+  - Facilitates the conversation.
+  - Ensures everyone is heard.
+  - Reminds scrums principles and sprint planning goals if necessary.
+- **[Product owner](https://www.agilealliance.org/glossary/product-owner/)**: 
+  - Prior to the sprint planning: performs some [backlog grooming](../backlog-management/grooming/readme.md) to ensure that each story that they want to propose for the new sprint (*) :
+     - Is in the correct position in the backlog, by right priority order.
+     - Contains a clear description.
+     - Contains a clear [definition of done](https://www.agilealliance.org/glossary/definition-of-done/).
+     - Do NOT pre assigns stories to the future sprint. This is the purpose of the sprint planning.
+  - During the meeting: 
+    - Describes to the team the stories that they propose for the sprint.
+-  **All team members**:
+   - Listen to the product owner story description.
+   - Ask questions to make sure everyone understands each story properly.
+   - [Estimate](./estimation/readme.md) the effort for each backlog item, as a team. 
+   - Split each story into tasks.
+   - (Optional) self assign first task to team members.

```

Comment:
In addition to optional assignment, another bullet point would be helpful to prioritize task list so that the team can pick up next available task based on the priority when they are done with existing task.